README.md

```diff
@@ -67,13 +62,16 @@ fn main() {
 }
 ```
 
-this will show a nice, colorful diff in your terminal, like this:
+... which has the benefit to show a nice, colorful diff in your terminal,
+like this:
 
 ```diff
 -1337
 +92
 ```
 
+More detailed information is available in the [documentation]. :-)
```

Comment:
I don't know if it's in the markdown specs, but github and my atom IDEs markdown preview treat it case-insensitive:
https://github.com/colin-kiegel/assert_cli/blob/1f29f4f25978bedf0bf476f5a27bedb0ea6a83e5/README.md